packages/__examples__/.storybook/stories/stories.ts

```diff
@@ -107,5 +108,11 @@ console.log(
     }
   }
 )
+
+// Stories not tied to one component
+storiesOf('Tooltip positioning', module).add('Tooltip positions', () =>
+  TooltipPositioning()
+)
+numStories++
```

Comment:
This is probably good for now, but let's try to write code that adds these extra stories automatically later (once we have more). 👍🏻